src/screens/OtherProfile/hooks/useOtherProfileScreenHooks.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import SimpleToast from 'react-native-simple-toast';
 
 import StorageUtils from '../../../utils/storage';
+import useDiscovery from '../../DiscoveryScreenV2/hooks/useDiscovery';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new import `useDiscovery` from `'../../DiscoveryScreenV2/hooks/useDiscovery'` is added. Ensure that this hook is properly tested and does not introduce side effects when used in this context.
```diff
+ import useDiscovery from '../../DiscoveryScreenV2/hooks/useDiscovery';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->